### DISCUSSION TITLE
I don't know how to make a pull request. I'm an amateur・

### DISCUSSION THREAD

**Discussion by nbvrivnf** [community/community#58417] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

The pull request method is not working properly. Doing what the internet says doesn't work. I forked someone else's repository, edited it locally, and pushed it, but it doesn't work. I will paste the url, so please advise.

url
https://github.com/nbvrivnf/tailwindcss-and-nextjs-portfolio/tree/nbvrivnf

 It's a branch called nbvrivnf.

**Comment by the-officialjosh** [comment]
Based on what I can see, it appears that your code is two commits ahead of the main repository. This means that when you push your changes directly to the repository, it works as expected. However, the issue arises when using the pull request method.

Pull requests are typically used to propose changes to a repository and ask the owner or collaborators to review and merge them into the main project. It's important to note that only the owner of the repository or selected collaborators has the authority to merge pull requests.

In this case, since you're trying to make changes to someone else's repository, creating a pull request will not automatically update the original project. Pull requests serve as a way to request the repository owner or collaborators to review your changes and decide whether to merge them.

For your situation, if you intended to use the repository for your own purposes without contributing back to the original project, it would have been more appropriate to clone the repository instead of forking it. Cloning allows you to have a local copy of the repository that is not connected to the original project, and you can make changes to it without affecting the original repository.

If you have already pushed changes to your forked repository, it is possible to reset it to match the original repository. However, please keep in mind that resetting will remove your modifications from the forked repository.

To summarize, when using pull requests, only the repository owner or designated collaborators can merge the changes. If you have any further questions or need assistance, feel free to ask.

**Reply by nbvrivnf** [reply]
Thank you for your response. I will give it a try. If I have any further questions, I appreciate your assistance.

**Reply by nbvrivnf** [reply]
I apologize for the inconvenience you're facing. To better understand the situation and provide you with an accurate solution, could you please provide more details about the error you encountered? Specifically, any error messages or information you received when trying to push your changes? Additionally, if you could let me know the version control system or platform you are using (e.g., Git, GitHub, Bitbucket), it would be helpful in determining the possible causes and solutions.



url

https://github.com/nbvrivnf/tailwindcss-nextjs-portfolio

**Reply by airtower-luna** [reply]
That repository isn't a fork, so you won't be able to make PRs to any other repository. You should be able to make PRs to merge branches within your repository.

**Reply by nbvrivnf** [reply]
This is another repository. I tried cloning as advised, but something went wrong with the first push, so I pushed again, but it didn't deploy properly.

**Reply by the-officialjosh** [reply, marked as answer]
Great job on pushing the code properly this time! In order to assist you more effectively, please provide answers to the following questions:

1. Is the program running well on your localhost after you did `npm install` and all?

2. From what I could see, there is a red "X" close to all commits you have made so far, indicating a Vercel deployment issue as you mentioned. However, I cannot see the specific error as I am not a collaborator on your project. You might want to click on the red "X" to access Vercel, where you will find the exact issue preventing the deployment. If I were to guess, it could be related to an unused import. If you are unsure how to resolve it, please share the error here, and I will guide you through the resolution process. It should be an easy fix.

Please provide the requested information, and I'll be happy to assist you further.

**Reply by nbvrivnf** [reply]
I did npm install and it works fine locally.


Failed to compile.
./components/HeroSection.tsx
26:32  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities
./components/SlideUp.tsx
29:6  Warning: React Hook useEffect has a missing dependency: 'offset'. Either include it or remove the dependency array.  react-hooks/exhaustive-deps
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1
BUILD_UTILS_SPAWN_1: Command "yarn run build" exited with 1
----------------------------------------------------------------------------

User
./components/SlideUp.tsx
29:6  Warning: React Hook useEffect has a missing dependency: 'offset'. Either include it or remove the dependency array.  react-hooks/exhaustive-deps
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1
BUILD_UTILS_SPAWN_1: Command "yarn run build" exited with 1

**Reply by nbvrivnf** [reply]
Deployed! ! ! !
thank you! ! !
First, I didn't build the project locally, and I was able to rewrite the contents of .eslintrc.json. Thanks for all the advice! ! !

**Reply by the-officialjosh** [reply]
Please can you mark that answer earlier as the answer for those with similar issues

**Reply by nbvrivnf** [reply]
thx
